app-text/asciidoc/asciidoc-9999.ebuild

```diff
@@ -50,6 +50,10 @@ If you are going to use a2x, please also look at a2x(1) under
 REQUISITES for a list of runtime dependencies.
 "
 
+# waiting to be merged upstream, see
+# https://github.com/asciidoc/asciidoc/pull/91
+PATCHES=( "${FILESDIR}/asciidoc-9999_fix_readme_rule.patch" )
```

Comment:
To be honest, I don't like the idea of using patches on live ebuild. But I see that upstream slacks a lot…